runtime/autoload/netrw.vim

```diff
@@ -24,8 +24,8 @@
 "   2024 Aug 22 by Vim Project: fix mf-selection highlight (#15551)
 "   2024 Aug 22 by Vim Project: adjust echo output of mx command (#15550)
 "   2024 Sep 15 by Vim Project: more strict confirmation dialog (#15680)
-"   2024 Sep 19 by Vim Project: mf-selection highlight uses wrong pattern (#15700)
```

Comment:
Mistake? (Or, needs to be rebased?)

Reply:
Somehow got lost in between, readded. Thank you for your input!